x-pack/plugins/lists/common/schemas/request/create_list_item_schema.ts

```diff
@@ -18,7 +18,7 @@ export const createListItemSchema = t.intersection([
       value,
     })
   ),
-  t.exact(t.partial({ id: idOrUndefined, meta: metaOrUndefined })),
+  t.exact(t.partial({ id, meta })),
```

Comment:
NOTE: You see all these changes because I did not need the `OrUndefined` as the `RequiredKeepUndefined` works like magic and keeps it to where I can force `undefined` when I need/want to vs. not.

This makes things look like a more clean/clear io-ts implementation.